app/move/controllers/create/save.test.js

```diff
@@ -554,6 +554,20 @@ describe('Move controllers', function() {
             }
           )
         })
+
+        context('when to location is also Prison', function() {
+          beforeEach(function() {
+            req.form.values.to_location_type = 'prison'
+            controller.process(req, {}, {})
+          })
+          it('will create a status of proposed on the move', function() {
+            expect(
```

Comment:
This would be easier to read as:

```javascript
expect(req.sessionModel.set).to.be.calledOnceWithExactly(
  'status',
  'proposed'
)
```

And you then don't need to include `isEqual`. Feels unnecessary as mocha already handles assertions.

Or if you need to assert a particular call (where the argument to `getCall` is the call number using zero based index):

```javascript
expect(req.sessionModel.set.getCall(1)).to.be.calledWithExactly(
  'status',
  'proposed'
)
```

Reply:
This was done to prevent the test to be unnecessarily bound by order of calls; but fair point re: readability.